p2panda-rs/src/lib.rs

```diff
@@ -45,7 +45,7 @@
 #![warn(
     missing_copy_implementations,
     missing_debug_implementations,
-    missing_doc_code_examples,
+    rustdoc::missing_doc_code_examples,
```

Comment:
This is trickier: https://github.com/p2panda/p2panda/issues/120

Reply:
Ah! I thought there was a complication there, but it worked in the library, so i figured it was ok... I reverted it now :+1: